proposals/software-ecosystem-map.md

```diff
@@ -0,0 +1,74 @@
+### Title
+
+<!-- A short, pithy title for the proposal. -->
+
+Long-term open source strategies: Software ecosystem map
+
+### Short description
+
+<!-- A short, one-sentence description of the proposal. -->
+
+To conduct a dedicated study for the software ecosystem that shows the resource allocation on proprietary and open source software.
+
+### Author(s)
+
+<!-- Put your GitHub username(s) here. The proposal author(s) will "own" the proposal and will be able to accept future PRs that change it. -->
+
+[@coni2k](https://github.com/coni2k)
+
+
+### Proposal body
+
+<!-- Explain your proposal. Add as much as you want, within reason! -->
+
+**Intro**
+
+At the macro level, open source software has a significantly higher return on investment than proprietary software, both in commercial and non-commercial settings.
+
+However, OpenForum Europe's [latest study](https://digital-strategy.ec.europa.eu/en/library/study-about-impact-open-source-software-and-hardware-technological-independence-competitiveness-and) indicates that around 99 percent of the software is still developed as proprietary software.
+
+We suggest governments tackle this issue as a [market failure](https://www.investopedia.com/terms/m/marketfailure.asp) and develop long-term strategies to increase open source software adoption, such as FSFE's [Public Money, Public Code](https://publiccode.eu/) campaign.
+
+To measure the effectiveness of these strategies, we first need to map the software ecosystem and use it as one of the Key Performance Indicators.
+
+Keeping track of this map's changes would allow us to estimate the required time and resources to achieve specific targets.
+
+**Goals**
+
+The study should show the resource allocation (capital, engineering) on proprietary and open source software, at least in the following areas:
```

Comment:
Interesting idea! I'm curious as to how you plan to collect this data? I'd imagine a lot of this data might be internal / not available publicly.

Reply:
After your question, I wanted to check the Worldwide Developer Population study from Evans Data:
https://evansdata.com/reports/viewRelease.php?reportID=9

"The methodology used by Evans Data Corporation to establish the size and composition of the developer population consists of several phases. EDC calculated the actual population figures by using results from the Global Development Survey, which was filtered with respect to specific areas of interest, cross-tabbed with the four regions — North America; Asia-Pacific (APAC); Europe, the Middle East and Africa (EMEA); and Latin America — and then combined with the current results yielded from EDC's propriety predictive model."

So, it looks like, a periodic survey combined with their statistical model. Can they extend their study by including the spendings for open source software? It would be great to ask them.

On the other hand, maybe their report already has this data, but I didn't consider contacting their sales 🤷‍♂️ (There was a report from another company that was around $3000).

What I did before is to look into the Open Source Contributor Index website and find the sum of all "Active Contributors" (10+ commits):
https://opensourceindex.io/

There were around 50,000 developers in a month. Comparing this number to the global population (~27 million) indicates that we invest less than 1% of our engineering resources in open source software.

But this is only the global number. It would be great to see the public sector in Europe to see the effect of Public Money, Public Code campaign for instance.

Again, my first approach would be to contact these research organizations. If they already have the data, can we make it public? If not, can they include it in their following study?

Reply:
Thanks!